designs/alert_grouping.md

```diff
@@ -0,0 +1,19 @@
+# Grouping alerts from a cluster
+
+Author: achvatal
+
+Last updated: 26 Apr
+
+## Goals
+
+Limit the number of alerts from clusters so that a single root problem on a cluster should only generate a single alert.
+
+## Current state
+
+Every critical alert generates a Pager Duty incident, regardless of timing
+
+## Proposal
+
+Enable time-based alert grouping on Pager Duty services to automatically merge new incidents into an existing incident (for the same service) if its most recent alert is less than an hour old.
```

Comment:
Thoughts on making this 1 hour duration configurable (I'm not sure how confident we are that 1 hour is enough :D)? In that case this would be a new optional field on the CR as well.